files/autopause/autopause-daemon.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+exec 1>/tmp/terminal-mc
```

Comment:
I'm curious what this is doing?

Reply:
redirects stdout to the terminal, that is logged by docker, as this script is called detached